src/packages/pentestkit/controller/rbuilder.js

```diff
@@ -26,7 +26,7 @@ export class ptk_controller_rbuilder {
         return browser.runtime.sendMessage({
             channel: "ptk_popup2background_request",
             type: "update_request",
-            item: item
+            item: JSON.parse(JSON.stringify(item))
```

Comment:
is this need, you are turning them item into a string then back to the object? is this to do a deep copy?